arches/app/utils/request.py

```diff
@@ -1,7 +1,6 @@
 def looks_like_api_call(request):
     known_api_paths_without_api_signposting = [
         # Until these are renamed for consistency, just list them.
-        "arches.app.views.workflow_history",
```

Comment:
This is the only one I intentionally moved, but there could be more that deserve moving.